Tests/Security/TwoFactor/Provider/Google/GoogleAuthenticatorTest.php

```diff
@@ -95,25 +45,9 @@ public function getHostnameAndIssuerToTest()
     {
         return [
             [null, null, 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth%3A%2F%2Ftotp%2FUser%2520name%3Fsecret%3DSECRET'],
-            ['Hostname', null, 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth%3A%2F%2Ftotp%2FUser%2520name%40Hostname%3Fsecret%3DSECRET'],
-            [null, 'Issuer Name', 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth%3A%2F%2Ftotp%2FIssuer%2520Name%3AUser%2520name%3Fsecret%3DSECRET%26issuer%3DIssuer%2520Name'],
-            ['Hostname', 'Issuer Name', 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth%3A%2F%2Ftotp%2FIssuer%2520Name%3AUser%2520name%40Hostname%3Fsecret%3DSECRET%26issuer%3DIssuer%2520Name'],
+            ['Hostname', null, 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=otpauth%3A%2F%2Ftotp%2FUser%2520name%2540Hostname%3Fsecret%3DSECRET'],
```

Comment:
I kept these tests to be sure that the modification of the deps does not lead to a regression.
I will update these tests.

Reply:
Yea, makes sense from that perspecitive.

Since we have seen it working and we're going the extract the creation of TOTP out, I think we'd be better off by removing it from the test class (and the other one) and have a single integration test, which is checking the interaction between the bundle and the TOTP class.

Or we could have a `GoogleAuthenticatorIntegrationTest`, which is testing with a real (not a mocked one) TOTP instance.